requirements/tox.txt

```diff
@@ -0,0 +1 @@
+tox==3.9.0
```

Comment:
OEP-18?

Reply:
I agree, but I would prefer to do this in a separate PR, to avoid too many moving parts and make this PR more manageable

Reply:
I'm good with this. Should probably add a ticket

Reply:
https://openedx.atlassian.net/browse/TE-2905